backend/LeetWars.Core/LeetWars.Core.Common/DTO/Challenge/BriefChallengeInfoDto.cs

```diff
@@ -0,0 +1,14 @@
+﻿using LeetWars.Core.Common.DTO.User;
+using LeetWars.Core.DAL.Entities;
+
+namespace LeetWars.Core.Common.DTO.Challenge
+{
+    public class BriefChallengeInfoDto : AuditEntity<long>
```

Comment:
You did not use audit entity's props on frontend so you can use Entity<T> here instead, which just keeps the id property of an entity. And you also do not need the ChallengeId since the Id field is given by both Entity and AuditEntity